### ISSUE TITLE
angular5 angular-cli compilation error

### ISSUE THREAD

**Issue by kandangit** [matiboy/angular2-prettyjson#25]
Hi, 
In an Angular 5 project setup with angular-cli:
Having the following compiling error when compiling a project that uses prettyjson dependency in the package.json:

```
> ERROR in Error during template compile of 'PrettyJsonModule'
>   Function expressions are not supported in decorators in 'NgModule'
>     'NgModule' contains the error at @angular\core\core.ts(194,31)
>       Consider changing the function expression into an exported function.
```

	  
package.json includes the dependency:	  
```
"dependencies": {
    "angular2-prettyjson": "^2.0.6",
	...	
}
```

Looks like node_modules in the angular2-prettyjson folder gives a conflict with the main angular 5 version of my project (Angular 5.1.0).
Any idea what is the  reason? Is this not compatible with Angular 5? thanks.

**Comment by ethanve** [comment]
Having similar issues here

**Comment by matiboy** [comment]
Hi @kandangit & @ethanve 
Thank you for opening this issue.
Which version of Angular are you using? There seemed to be a similar issue #24 which was resolved when Angular 5.1.2 came out

Please let me know whether upgrading fixes it for you

Mat

**Comment by kandangit** [comment]
Hi @matiboy 

Thanks a lot for your quick answer. I am using 5.1.0

Yes, it gets fixed with 5.1.2. Looks like node_modules folder within angular2-prettyjson/node_modules is not created with 5.1.2 and does not cause compiling issues.

Sorry as searched through google and didn´t find the old #24 issue.

Regards

**Comment by matiboy** [comment]
@kandangit No problem, glad it works again 👍
